src/xrEngine/xr_ioc_cmd.cpp

```diff
@@ -581,16 +581,11 @@ class CCC_renderer : public CCC_Token
 
         r2_advanced_pp = (renderer_value >= 3);
     
-        cmd_lock = TRUE;
+        cmd_lock = true;
     }
 
-    virtual void Save(IWriter* F)
+    void Save(IWriter* F) override
     {
-        if (renderer_allow_override == FALSE)
-        {   // Do not save forced value
-            return;
-        }
```

Comment:
Ah, why it was discarded? I found it very nice!

So, the final approach: can you return it and squash the commits? (although, squashing isn't necessary, this can be done with merge-button click)